src/graphql/validation/specified_rules.py

```diff
@@ -118,15 +118,15 @@
         PossibleFragmentSpreadsRule,
         NoFragmentCyclesRule,
         UniqueVariableNamesRule,
-        NoUndefinedVariablesRule,
-        NoUnusedVariablesRule,
+        # NoUndefinedVariablesRule,
+        # NoUnusedVariablesRule,
```

Comment:
This should be reverted

Reply:
See below, the default rules should not be changed.